tests/mobly/logger_test.py

```diff
@@ -23,8 +24,10 @@ class LoggerTest(unittest.TestCase):
     """Verifies code in mobly.logger module.
     """
     def test_epoch_to_log_line_timestamp(self):
-        # actual_stamp = logger.epoch_to_log_line_timestamp(1469134262116)
-        # self.assertEqual("07-21 13:51:02.116", actual_stamp)
+        test_time = time.localtime(1469134262)
+        correct_stamp = time.strftime("%m-%d %H:%M:%S.116", test_time)
```

Comment:
Seems strange to encode '116' right into the format string. Stack Overflow suggests to get milliseconds into the formatted string like so:
http://stackoverflow.com/a/18406412

Perhaps you could use this instead?